sample/Pizzeria.Store.Web/Services/IPizzeriaApiClient.cs

```diff
@@ -17,5 +17,3 @@ public interface IPizzeriaApiClient
     [Post("/customers")]
     Task CreateCustomerAsync([Body] CreateCustomerRequest request, CancellationToken cancellationToken = default);

```

Comment:
The Blazor app no longer needs to call this Store API endpoint directly so it can be removed